kommunicate/src/main/java/com/applozic/mobicomkit/api/account/register/RegisterUserClientService.java

```diff
@@ -202,6 +202,10 @@ public RegistrationResponse createAccount(User user) throws Exception {
     }
 
     public boolean refreshAuthToken(String applicationId, String userId) {
+        MobiComUserPreference mobiComUserPreference = MobiComUserPreference.getInstance(context);
```

Comment:
@pahadi777 Did you test the api by passing one parameter ?As of now we have 
1. applicationid
2. userid
3. device key
Just test by passing only one param for  each & check what's happening

Reply:
Passing any parameter as null is giving failure on calling refreshAuthToken . Only passing all the parameters is giving success .